pandas/tests/arrays/test_timedeltas.py

```diff
@@ -51,6 +63,17 @@ def test_copy(self):
 
 
 class TestTimedeltaArray(object):
+    def test_np_sum(self):
+        # GH#25282
+        # TODO: result should be a Timedelta, not a np.timedelta64 object
+        vals = np.arange(5).view('m8[h]').astype('m8[ns]')
```

Comment:
why both view and astype? (one of both should be sufficient?)

Reply:
The `astype` is necessary since TimedeltaArray only accepts m8[ns].  The view is personal preference to pass data thats not just single-digit-nanos.

Reply:
ah, I missed the different in unit